src/contracts/pods/EigenPodManager.sol

```diff
@@ -279,6 +287,25 @@ contract EigenPodManager is
     }
 
     // VIEW FUNCTIONS
+    /// @notice Return the real number of ETH shares that the `podOwner` has
+    function podOwnerShares(address podOwner) public view returns (int256) {
+        int256 podOwnerShares;
+        if (podOwnerScaledShares[podOwner] < 0) {
+            podOwnerShares = -int256(
```

Comment:
should we just add this gross stuff to the library?

Reply:
Probably, there is a lot of gross casting and checks

Reply:
Will leave this for later